templates/landscape/pricing.html

```diff
@@ -105,16 +105,16 @@ <h1>Choose the best Landscape for you</h1>
         </tr>
         <tr>
           <td>Pricing</td>
-          <td>Landscape SaaS is included with a paid Ubuntu Pro subscription.</td>
+          <td>Landscape SaaS is included with any Ubuntu Pro subscription.</td>
           <td>Free for up to 10 machines for personal use, or evaluation purposes.<br /><br />Beyond the free tier, <a href="/pro">Ubuntu Pro</a> is required for more than 10 machines on self-hosted Landscape</td>
           <td>Canonical's standard <a href="/managed/apps">Managed Apps</a> pricing applies</td>
         </tr>
         <tfoot>
           <tr>
             <td></td>
-          <td><a href="/pro/subscribe">Pricing calculator for Ubuntu Pro</a></td>
-            <td><a href="/landscape/install">Set up your self-hosted Landscape</a></td>
-            <td><a href="/contact-us/form?product=landscape" class="p-button--positive is-dense js-invoke-modal">Contact us</a> to get started</td>
+            <td><a href="#get-in-touch" class="p-button--positive is-dense js-invoke-modal">Get started</a></td>
```

Comment:
```suggestion
            <td><a href="/contact-us/form?product=landscape" class="p-button--positive is-dense js-invoke-modal">Get started</a></td>
```
can we keep these as fallbacks in the case that JS is not enabled?